### ISSUE TITLE
Gradient overflow issue when i try to train gpt2 model in fp16 with 02. Any idea why that maybe happen?

### ISSUE THREAD

**Issue by niklaras** [NVIDIA/apex#864]
![Screenshot from 2020-05-31 23-38-12](https://user-images.githubusercontent.com/35643708/83362226-3a7f5000-a398-11ea-8b75-35f06eda4ba6.png)
i get this message every 2000 steps.
"Gradient overflow. Skipping step, loss scaler 0 reducing loss scale to 262144.0"
![Screenshot from 2020-05-31 13-49-05](https://user-images.githubusercontent.com/35643708/83350677-696cd600-a346-11ea-92f6-85e8940cfea0.png)

The train run on rtx 2070 and seems to not have other problem but that message keeps going on every 2000 steps. Is it a problem? 
pytorch 1.5.0
python 3.7
cuda 10.1

**Comment by ptrblck** [comment]
The loss scaler tries to increase the loss scaling factor after a threshold of successful steps was reached.
In your case it seems that the scaling factor is being downgraded to the same value, so it should be fine.

Closing this issue, but feel free to reopen, in case you encounter any issues.
